features/minion/heartbeat/producer/pom.xml

```diff
@@ -35,5 +35,23 @@
             <artifactId>org.opennms.features.minion.heartbeat.common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.opennms.features.api-layer</groupId>
+            <artifactId>org.opennms.features.api-layer.minion</artifactId>
+            <version>31.0.2-SNAPSHOT</version>
```

Comment:
Use `${project.version}` here and bellow.

Reply:
> Use `${project.version}` here and bellow.
Sorry. I missed that.